### PR TITLE
a HUMONGOUS sprite update(actually theft)

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

Oh my god I will perish while listing all of the changes:
cigar resprite, cigar case resprite, locked+open states, needs to be open to take a cigar, storage capacity reduced from 7 to 6(not nerf, for the sake of the sprite), sprites by CeUvi
Water and welder fuel tanks resprite - stolen from Tau Ceti, smol tanks by WatlerJe, big tanks by CeUvi
A fuckathonatonobig resprite of drinks, more unique glass states, all set to true manually - stolen from Tau Ceti
Fireaxe in hand, item, on back resprite - stolen from Tau Ceti
Flamethrower total resprite - stolen from Tau Ceti
Atmos cans, pumps and scrubbers resprite - stolen from Tau Ceti
Autopsy scanner resprite - stolen from Tau Ceti
Candles resprite - stolen from Tau Ceti
probably something else - stolen from Tau Ceti

!!__IMAGES IN COMMENTS__!! (Because git died when I was making this desc and I don't want to post-process it)
and they are also here
image
## Why It's Good For The Game

bay leftovers bad
old sprites bad
my opinion good
new sprites good
watlerJe gave consent :flushed:
## Changelog
:cl: Stolen from Tau Ceti(WatlerJe and the Gang), CeUvi, Kegdo
imageadd: Water and welder fuel tanks resprite -  smol tanks by WatlerJe, big tanks by CeUvi
imageadd: A fuckathonatonobig resprite of drinks, more unique glass states, all set to true manually
imageadd: Fireaxe in hand, item, on back resprite
imageadd: Flamethrower total resprite
imageadd: Atmos cans, pumps and scrubbers resprite
imageadd: Autopsy scanner resprite
imageadd: Candles resprite
imageadd: Lamps resprite
imageadd: probably something else
imageadd: cigar case and cigars
add: cigar case can be opened and closed, needs to be opened to access cigs, storage capacity reduced from 7 to 6 (sprite reasons not nerfing I swear)
/:cl:

### CODE REVIEW

**Comment by Omni44** [comment]
Grand Theft Spritework

**Comment by SaltyOldBull** [comment]
good work sprayters

**Comment by SirRichardFrancis** [comment]
It's cool. But please provide screenshots for sprite-changing PRs, at least show new sprites.

**Comment by Kegdo** [comment]
I just woke up, gimme a second.

**Comment by Kegdo** [comment]
image
image
image
image
image
image
image
cigar2 replaced with just cigar
image
image
image
image

**Comment by Kegdo** [comment]
image
image
image
image
image
heater^

**Comment by Kegdo** [comment]
somethin' weird with drinks, checking right now

**Comment by Kegdo** [comment]
nvm they all cool
image
image
image
lotsa animated

**Comment by CeUvi** [comment]
their drinks look awful
also remove syndiebomb, we have resprited glass for it

**Comment by Kegdo** [comment]
ours are mostly bay leftovers, so both are shit, but tau is a bit better

**Comment by Kegdo** [comment]
> their drinks look awful also remove syndiebomb, we have resprited glass for it

image
no?

**Comment by Kegdo** [comment]
the GLASS DRINK sprite is shit, the double glass thing is untouched.

**Comment by CeUvi** [comment]
> the GLASS DRINK sprite is changed, the double glass thing is untouched.

just pour the drink in the double glass

**Comment by SirRichardFrancis** [comment]
Oddly enough, I agree with CeUvi, we have our own drink sprites coming, no need for port. Also bandage sprite isn't better than our current one, imo even worse. Other than that, looks great.

**Comment by Kegdo** [comment]
> we have our own drink sprites coming, no need for port.

oh, we do? You mean, those drink sprites that were requested, like, half a year ago? Yeah sure. Mmhm. Yup. Of course. They will be on eris soon, I am 100% sure. But, until the said drinks are actually sprited, let's replace the ugly bay leftovers with tau sprites. They are not as horrible.

> Also bandage sprite isn't better than our current one, imo even worse.

I know, I'll try to find some more sprites today to replace 'em.

**Comment by CeUvi** [comment]
> They are not as horrible.

they are even more horrible than bay leftovers.

**Comment by Kegdo** [comment]
> > They are not as horrible.
> 
> they are even more horrible than bay leftovers.

said the person who likes his own BP helmet resprite and small cigars.

**Comment by Kegdo** [comment]
image
I refuse to believe that someone can UNIRONICALLY think that the sprites on the left are better than the ones on the right

**Comment by CeUvi** [comment]
> said the person who likes his own BP helmet resprite and small cigars.

said the person who doesn't listen to authorities and he's not even a spriter

**Comment by Kegdo** [comment]
> > said the person who likes his own BP helmet resprite and small cigars.
> 
> said the person who doesn't listen to authorities and he's not even a spriter

My opinion is more important 😎

**Comment by CeUvi** [comment]
> My opinion is more important 😎

ok, MINER.
minor, I'd say.

**Comment by Kegdo** [comment]
image

**Comment by CeUvi** [comment]
cigcase
minicigs supremacy

**Comment by Kegdo** [comment]
the spoon shall be your ultimate demise

**Comment by Kegdo** [comment]
image
that's it, now it's time to go to bed till friday
unless...?

**Review by infraredbaron — APPROVED** [review]
Sprites are good. Even less bay12 leftovers remain.

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -286,18 +286,83 @@
...
+	set name = "Close cigar case"
+	set category = "Object"
+	set src in view(1)
+	if(!is_worn())
+		if(can_interact(user) == 1)	//can't use right click verbs inside bags so only need to check for ablity
+			return
+
+		open_close(user)
+	else
+		to_chat(user, SPAN_NOTICE("You can\'t open \the [src] while it\'s equipped!"))
```
Comment (hyperioo) [inline code/game/objects/items/weapons/storage/fancy.dm:321]:
```suggestion
		to_chat(user, SPAN_NOTICE("You cannot open \the [src] while it\'s equipped!"))
```

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 2]
```diff
@@ -286,18 +286,83 @@
...
+		if(able == 1)
+			return
+
+		if(able == 2)
+			to_chat(user, SPAN_NOTICE("You cannot open \the [src] while it\'s in a container."))
+			return
+
+		open_close(user)
+	else
+		to_chat(user, SPAN_NOTICE("You can\'t open \the [src] while it\'s equipped!"))
```
Comment (hyperioo) [inline code/game/objects/items/weapons/storage/fancy.dm:336]:
```suggestion
		to_chat(user, SPAN_NOTICE("You cannot open \the [src] while it\'s equipped!"))
```

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 3]
```diff
@@ -286,18 +286,83 @@
...
+			w_class = ITEM_SIZE_SMALL
+			open = TRUE
+		else
+			to_chat(user, SPAN_NOTICE("You close \the [src]."))
+			w_class = ITEM_SIZE_TINY
+			open = FALSE
+		playsound(loc, 'sound/machines/click.ogg', 100, 1)
+		update_icon()
+	else
+		to_chat(user, SPAN_NOTICE("You can\'t open \the [src] while it\'s equipped!"))
```
Comment (hyperioo) [inline code/game/objects/items/weapons/storage/fancy.dm:352]:
```suggestion
		to_chat(user, SPAN_NOTICE("You cannot open \the [src] while it\'s equipped!"))
```

**Pull request merged**